### PR TITLE
Add course settling alert icon

### PR DESCRIPTION
**Pull request by bnehoran**
image

Resolves: #124

### CODE REVIEW

**Comment by danielleung27** [comment]
Looks good!

**Review by danielleung27 — APPROVED** [review]

File: frontend/src/Requirements.css [hunk 1]
```diff
@@ -82,4 +82,8 @@
 
 .fa-search:hover {
   color: #e74c3c;
-}
\ No newline at end of file
+}
+
+.fa-exclamation-circle{
+  color: #c13424;
```
Comment (churichard) [inline frontend/src/Requirements.css]: We're using a standard red already: #e74c3c

File: frontend/src/Requirements.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ export function populateReqTree(reqTree){
                   return(<li key={index} className='settled' onClick={(e)=>{toggleSettle(course, requirement['path_to'], false)}}>{course}</li>);
                 })}
                 {requirement['unsettled'].map((course, index)=>{
-                  return(<li key={index} className='unsettled text-muted' onClick={(e)=>{toggleSettle(course, requirement['path_to'], true)}}>{course}</li>);
+                  return(<li key={index} className='unsettled text-muted' onClick={(e)=>{toggleSettle(course, requirement['path_to'], true)}}>{course} <i class="fa fa-exclamation-circle" title="This course could satisify multiple requirements. Click to settle it here."></i></li>);
```
Comment (churichard) [inline frontend/src/Requirements.js]: You have to hover over the icon and leave your mouse there for a little bit before the text pops up. Do you think we should have it pop up immediately on hover?

**Review by churichard — CHANGES_REQUESTED** [review]

File: frontend/src/Requirements.js (hunk 2 above)
Comment (churichard) [reply frontend/src/Requirements.js]: Also, it should be `className`, not `class`.

File: frontend/src/Requirements.css (hunk 1 above)
Comment (bnehoran) [reply frontend/src/Requirements.css]: Changed

File: frontend/src/Requirements.js (hunk 2 above)
Comment (bnehoran) [reply frontend/src/Requirements.js]: Switched to `className`

**Review by churichard — APPROVED** [review]

**Review by churichard — CHANGES_REQUESTED** [review]
Still needs some work, tooltips aren't quite ready yet.

**Review by churichard — APPROVED** [review]

**Pull request merged**